string device
string EthIPAddr
string MacAddress
uint32 PacketCount
uint8 ChargeState
uint16 Flags
uint16 PowerLevel
float32 VMon3v3
float32 VMon5v
float32 IMon5v
float32 VMon12v
float32 IMon12v
float32 VMonGateDriver
float32 IMonGateDriver
float32 VMonPA
float32 IMonPa
float32 TMonPa
float32 VMonBatt
float32 VMonBattProg
float32 VRect
float32 TBoard
float32 ICharger
float32 IBattery
float32 TargetIBatt
float32 IMaster
float32 ISlave1
float32 ISlave2